tests/AzureStorageContextTest.php

```diff
@@ -160,7 +162,7 @@ public function testShouldAllowDeleteTopic()
             ->method('deleteQueue')
```

Comment:
`deleteTopic` ?

Reply:
Nope. `AzureStorageContext` invokes `deleteQueue` method of `QueueRestProxy`, because storage queues do not have concept of topics.